manifests/client.pp

```diff
@@ -9,6 +9,6 @@
   contain ::mcollective::client::install
   contain ::mcollective::client::config
 
-  Class['mcollective::client::install'] ->
-  Class['mcollective::client::config']
+  Class['mcollective::client::install']
+  -> Class['mcollective::client::config']
```

Comment:
we agreed somewhere in the IRC to go with the new puppet-lint style. This is recommended by the style guide. We should discuss the style guide if we don't like it, but not disable the linter.